installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh

```diff
@@ -40,7 +55,50 @@ do
     service_name=$element
   fi
 
-  if [ $element == "jazz_email" ] || [ $element == "jazz_usermanagement" ] || [ $element == "jazz_codeq" ] || [ $element == "jazz_metrics" ]; then
+  if [[ $(inArray "${lambda_services[@]}" "$element") ]]; then
+	 if [[ $(inArray "${runtime_six_services[@]}" "$element") ]]; then
+	  	aws dynamodb put-item --table-name $tablename --item '{
+			  "SERVICE_ID":{"S":"'$uuid'"},
```

Comment:
Since you have taken the effort to make it easier for someone to add their services, I would take it one more step further and declare two variables outside this loop: `type` and `runtime`. With the four combinations we are trying to determine what `type` it is between `function | api` and what `runtime` it is: `nodejs4.3 | nodejs6.10`. Once we have those variable values you can just do   `aws dynamodb put-item --table-name $tablename --item '{ ...}'` at the end. Let me know if that makes sense.

Reply:
done